salt/grains/metadata.py

```diff
@@ -8,26 +8,35 @@
 To enable these grains that pull from the http://169.254.169.254/latest
 metadata server set `metadata_server_grains: True` in the minion config.
 
+Optionally set 'metadata_server_host: <metadata server host IP/FQDN, 
+e.g. TencentCloud metadata server is metadata.tencentyun.com>' to spec-
+ify the custom metadata server.
+
 .. code-block:: yaml
 
     metadata_server_grains: True
-
+    metadata_server_host: metadata.tencentyun.com
 """
 
+# Import python libs
 import os
 import socket
+from telnetlib import IP
 
+# Import salt libs
 import salt.utils.data
 import salt.utils.http as http
 import salt.utils.json
 import salt.utils.stringutils
 
-# metadata server information
-IP = "169.254.169.254"
-HOST = "http://{}/".format(IP)
```

Comment:
I think we maybe keep these here, but if you want you could probably initialize them as empty strings